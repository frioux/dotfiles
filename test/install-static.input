#!/bin/bash

dir='test/,install-static'

make --always-make >/dev/null \
  "INSTALLATION_DIR=$dir" \
  'TARGETS_STATIC=Makefile README' \
  install

find "$dir"

rm -rf "$dir"

# __END__
