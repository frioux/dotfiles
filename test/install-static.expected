test/,install-static
test/,install-static/Makefile
test/,install-static/README
